ignite/metrics/accumulation.py

```diff
@@ -25,13 +25,13 @@ class VariableAccumulation(Metric):
         - `+batch_size` if input is a ND `torch.Tensor`. Batch size is the first dimension (`shape[0]`).
```

Comment:
Let's remove all files related to metrics package :)

Reply:
I am expecting `ignite.metrics` to merge first, then this
But, ok I will remove metrics files here